recipes/natural_language_processing/rag/Makefile

```diff
@@ -16,7 +16,7 @@ run:
 # rag requires custom bootc because it uses an extra build-arg for CHROMADB_IMAGE (other apps use ../../common/Makefile.common target)
 .PHONY: bootc
 bootc: quadlet
-	podman build \
+	"${CONTAINER_TOOL}"  build \
```

Comment:
Did we want to include the `--layers=false` in this bootc command?

Reply:
We are not squashing so no.